spec/requests/api/v1/settings_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+describe 'GET api/v1/settings/need_update', type: :request do
+  let(:params) do
+    {
+      device_version: '1.0'
+    }
+  end
+  subject { get need_update_api_v1_settings_path, params: params, as: :json }
+
+  # Clean settings in case you have ran seeds
+  before { Setting.destroy_all }
+  after { Setting.destroy_all }
+
+  context 'with correct settings' do
+    let!(:setting_version) { create :setting_version }
+
+    it 'return success' do
+      subject
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'returns no need to update' do
+      subject
+      expect(json['must_update']).not_to be
+    end
+  end
+
+  context 'with incorrect settings' do
+    let!(:setting_version) { create :setting_version, value: '2.0' }
+
+    it 'returns it needs to update' do
+      subject
+      expect(json['must_update']).to be
```

Comment:
Isn't `false` necessary here?

Reply:
that matcher behaves like `be_truthy` one, but in this case due to method logic we should expect always a bool, will update thanks